data/102/0001/en_US.json

```diff
@@ -3,7 +3,7 @@
 		"0001": {
 			"name": "MAIL_ADDRESS_NOT_CONFIRMED",
 			"message": "",
-			"short_description": "Unknown cause.",
+			"short_description": "Unknown cause",
```

Comment:
Why?

Reply:
Entered something, realised it was wrong, re-entered placeholder, forgot the full stop